### PR TITLE
Add `AgeFractional` logical type 

### LINKED ISSUE #876: Add FractionalAge Logical Type
- As a user, I wish I could use Woodwork with a **FractionalAge** Logical Type ([1.5, 2.4, 3.5]).
- This should use the `np.float64` dtype as the Physical Type.

### LINKED ISSUE #1076: Age and AgeNullable are missing from the list of Logical Types with standard tags in the types and tags doc
The semantic tags section of Woodwork's types and tags guide lists all the Logical types that have standard tags. `Age` and `AgeNullable` should be added under the `numeric` section.

We can also consider changing that section so that it does not need to be updated every time the Logical Types change.

### PR DESCRIPTION
**Pull request by tamargrey**
- Creates an `AgeFractional` logical type that has the dtype `float64` and adds to relevant locations in Woodwork
- Closes #876, closes #1076 

Note: went with the name `AgeFractional` instead of `FractionalAge` to keep the age ltypes next to one another in alphabetical order

### CODE REVIEW

**Review by thehomebrewnerd — APPROVED** [review]
One minor thing, but otherwise looks good!

File: docs/source/guides/logical_types_and_semantic_tags.ipynb [hunk 1]
```diff
@@ -174,14 +174,22 @@
     "\n",
     "##### Age\n",
     "\n",
-    "Represents Logical Types that contain non-negative numbers indicating a person’s age.\n",
+    "Represents Logical Types that contain whole numbers indicating a person’s age.\n",
     "\n",
     "- **physical type**: `int64`\n",
     "- **standard tags**: `{'numeric'}`\n",
     "\n",
     "##### AgeNullable\n",
```
[the remark this answers is not in the document]
Comment (tamargrey) [reply docs/source/guides/logical_types_and_semantic_tags.ipynb]: whoopse, thanks for catching!

File: woodwork/serialize.py [hunk 2]
```diff
@@ -15,7 +15,7 @@
 )
 from woodwork.utils import _is_s3, _is_url
 
-SCHEMA_VERSION = '11.1.0'
+SCHEMA_VERSION = '11.1.1'
```
Comment (tamargrey) [inline woodwork/serialize.py:18]: @gsheni this good?

Comment (gsheni) [reply woodwork/serialize.py:18]: yep

**Review by gsheni — APPROVED** [review]

**Review by jeff-hernandez — APPROVED** [review]

**Pull request merged**